lumicks/pylake/kymotracker/kymoline.py

```diff
@@ -134,6 +134,11 @@ def seconds(self):
     def position(self):
         return self._image.to_position(self.coordinate_idx)
 
+    @property
+    def _has_defined_ends(self):
+        """Checks if beginning and end of the line are not in the first/last frame."""
+        return self.time_idx[0] > 0, self.time_idx[-1] < self._image.data.shape[1] - 1
```

Comment:
yeah I was originally gonna go with this, but thought it might be nice to return the `bool` for both start and stop. but I agree this makes more sense, and see next comment